<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>256</width>
    <height>209</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>251</width>
      <height>181</height>
     </rect>
    </property>
    <widget class="QWidget" name="pageMenu">
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>0</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Задание 1</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_5">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>120</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Задание 5</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_4">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>90</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Задание 4</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>60</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Задание 3</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>30</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Задание 2</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page1">
     <widget class="QLineEdit" name="lineEdit_a">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>50</y>
        <width>51</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_b">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>50</y>
        <width>51</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_Check">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>80</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Проверка</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelHint">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>231</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Введите 2 числа. Если сумма чисел
больше 10 и меньше 20 программа
покажет True</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelRes">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>120</y>
        <width>131</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_Back_1">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>140</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Back</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page2">
     <widget class="QPushButton" name="pushButton_Back_2">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>150</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Back</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelHint_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>231</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Введите 2 числа. Если оба числа
равны 10 или их сумма равна 10 программа
покажет True</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelRes_2">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>130</y>
        <width>131</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_b_2">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>60</y>
        <width>51</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_a_2">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>60</y>
        <width>51</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_Check_2">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>90</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Проверка</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page3">
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>111</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Your numbers:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>221</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_Back_3">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>140</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Back</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page4">
     <widget class="QLabel" name="labelHint_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>231</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Введите число. Программа проверит
является ли оно простым</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_Check_3">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>90</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Проверка</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_a_3">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>60</y>
        <width>51</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="labelRes_3">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>130</y>
        <width>131</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_Back_4">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>150</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Back</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page5">
     <widget class="QLabel" name="labelRes_4">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>120</y>
        <width>131</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_Back_5">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>140</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Back</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_a_4">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>50</y>
        <width>51</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="labelHint_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>231</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string> Ведите год, программа проверит
является ли он високосным.</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_Check_4">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>80</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Проверка</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>256</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
